system/lib/libc/musl/src/internal/pthread_impl.h

```diff
@@ -112,7 +112,7 @@ struct __timer {
 // XXX Emscripten: The spec allows detecting when multiple write locks would deadlock, so use an extra field
 // _rw_wr_owner to record which thread owns the write lock in order to avoid hangs.
 // Points to the pthread that currently has the write lock.
-#define _rw_wr_owner __u.__vi[3]
+#define _rw_wr_owner __u.__p[3]
```

Comment:
I'm not familiar enough with this code to know where to review the corresponding change - in which file is that?